test/unit/blots/inline.js

```diff
@@ -26,4 +27,68 @@ describe('Inline', function () {
       '<p><em>0</em><strong><em>12</em></strong><em>3</em></p>',
     );
   });
+
+  fdescribe('Strikethrough', function () {
+    it('renders attributes when strikethrough tag the first', function () {
+      const testHtml = `
+      <p>
+        <s style="background-color: #00ff00">BBB</s>
+        <span style="background-color: #00ff00">CCC</span>
+      </p>`.replace(/(\n|>\s+|\s+<)/g, (match) => match.trim());
```

Comment:
Let's move the RegExp to a constant to make code clearer.

Reply:
Agree with you